### PR TITLE
control-service: Fix release step in pipeline

### PR DESCRIPTION
**Pull request by murphp15**
# Why
The release step fails with the newer docker image version. But works with the previous setup.We need the newer version for java 17 support for other steps. 
# What
Revert the changes for that step only. 
# How has this been tested?
Tested on the previous commit and changes reverted to merge.

### CODE REVIEW

**Comment by tozka** [comment]
Why does it fail ?  Can you link and copy paste the failure? Can we fix it? 


But first let's merge this as I understand the main pipeline is red and fixing the main branch CICD pipeline this is highest priority

**Review by tozka — APPROVED** [review]

File: projects/control-service/cicd/.gitlab-ci.yml [hunk 1]
```diff
@@ -287,7 +287,13 @@ control_service_release:
 
 # TODO: automatically rebuild image on vdk-heartbeat change
 control_service_vdk_heartbeat_release:
-  extends: .images:dind
+  image: docker:19.03.15
```
Comment (tozka) [inline projects/control-service/cicd/.gitlab-ci.yml:290]: nit: write a comment that are are using older version because ...

**Comment by murphp15** [comment]
An example failure https://gitlab.com/vmware-analytics/versatile-data-kit/-/jobs/3658293978

File: projects/control-service/cicd/.gitlab-ci.yml (hunk 1 above)
Comment (murphp15) [reply projects/control-service/cicd/.gitlab-ci.yml:310]: Done

**Pull request merged**